sbt-plugin/src/main/scala/com/akkaserverless/sbt/AkkaserverlessPlugin.scala

```diff
@@ -43,7 +43,7 @@ object AkkaserverlessPlugin extends AutoPlugin {
 
   override def projectSettings: Seq[sbt.Setting[_]] = Seq(
     libraryDependencies ++= Seq(
-      "com.akkaserverless" % "akkaserverless-sdk-protocol" % "0.7.0-beta.19" % "protobuf",
+      "com.akkaserverless" % "akkaserverless-sdk-protocol" % "0.7.0-beta.19" % "protobuf-src",
```

Comment:
so this will solve the AnnotationsProto issue?

Reply:
Yes, I'm fairly confident it will (though there might be further problems lurking there, around flat/non-flat packages and naming clashes)